src/spec/prf_expand.rs

```diff
@@ -0,0 +1,71 @@
+use blake2b_simd::Params;
+
+const PRF_EXPAND_PERSONALIZATION: &[u8; 16] = b"Zcash_ExpandSeed";
+
+/// The set of domains in which $PRF^\mathsf{expand}$ is defined.
```

Comment:
```suggestion
/// The set of domains in which $PRF^\mathsf{expand}$ is defined.
/// This currently only includes domains used in Orchard.
/// TODO: refactor uses of $PRF^\mathsf{expand}$ in Sapling.
```

Reply:
I don't see a need to reference Sapling inside the `orchard` crate; instead this should be an issue over in the `librustzcash` repo.

Reply:
Opened https://github.com/zcash/librustzcash/issues/391.